service-discovery/stork-custom/src/test/java/io/quarkus/ts/stork/custom/StorkCustomLoadBalancerIT.java

```diff
@@ -0,0 +1,86 @@
+package io.quarkus.ts.stork.custom;
+
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.is;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.response.ValidatableResponse;
+
+import junit.framework.AssertionFailedError;
+
+@QuarkusScenario
+public class StorkCustomLoadBalancerIT {
```

Comment:
This scenario is part of `QUARKUS-1413` TD, right? I think we should make that clear somehow, perhaps a `@Tag`.